modules/ROOT/pages/indexes/indexes-for-full-text-search.adoc

```diff
@@ -0,0 +1,394 @@
+
+.Query a full-text index for a relationship property
+[source, cypher]
+----
+CALL db.index.fulltext.queryRelationships("communications", "meeting") YIELD relationship, score
+RETURN type(relationship), relationship.message, score
+----
+
+.Result
+[role="queryresult",options="header,footer",cols="3*<m"]
+|===
+| type(relationship) | relationship.message | score
+
+| "EMAILED" | "I have booked a team meeting tomorrow." | 0.3239005506038666
+
+3+d|Rows: 1
+
+|===
+
+To only obtain exact matches, quote the `STRING` you are searching for:
+
+.Query a full-text index for exact matches
+[source, cypher]
+----
+CALL db.index.fulltext.queryNodes("namesAndTeams", '"Nils-Erik"') YIELD node, score
+RETURN node.name, score
+----
+
+.Result
+[role="queryresult",options="header,footer",cols="2*<m"]
+|===
+| node.name | score
+| "Nils-Erik Karlsson" | 0.7588480710983276
+2+d|Rows: 1
+|===
+
+Lucene also allows the use of logical operators, such as `AND` and `OR`, to search for terms:
+
+.Query a full-text index using logical operators
+[source, cypher]
+----
+CALL db.index.fulltext.queryNodes("namesAndTeams", 'nils AND kernel') YIELD node, score
+RETURN node.name, node.team, score
+----
+
+.Result
+[role="queryresult",options="header,footer",cols="3*<m"]
+|===
+
+| node.name | node.team | score
+| "Nils-Erik Karlsson" | "Kernel" | 0.723090410232544
+3+d|Rows: 1
+
+|===
+
+It is possible to limit the search to specific properties, by putting the property name and a colon in front of the text being searched for.
+
+.Query a full-text index for specific properties
+[source, cypher, indent=0]
+----
+CALL db.index.fulltext.queryNodes("namesAndTeams", 'team:"Operations"') YIELD node, score
+RETURN node.name, node.team, score
+----
+
+.Result
+[role="queryresult",options="header,footer",cols="3*<m"]
+|===
+
+| node.name | node.team | score
+| "Nils Johansson" | "Operations" | 0.21363800764083862
+| "Maya Tanaka"   | "Operations" | 0.21363800764083862
+3+d|Rows: 2
+
+|===
+
+A complete description of the Lucene query syntax can be found in the link:https://lucene.apache.org/core/8_2_0/queryparser/org/apache/lucene/queryparser/classic/package-summary.html#package.description[Lucene documentation].
+
+
+[[text-array-properties]]
+=== Text array properties
+
+If the indexed property contains a text array, each element of this array is analyzed independently and all produced terms are associated with the same property name.
+This means that when querying such an indexed node or relationship, there is a match if any of the array elements match the query.
+For scoring purposes, the full-text index treats it as a single-property value, and the score will represent how close the query is to matching the entire array.
+
+For example, the following query matches the same node while referring to different elements in the array:
```

Comment:
```suggestion
```
I find this remark confusing. Also, I would alter the example's query string, for example with `late` rather than `work`, as `work` appears in all array entries and one might think that is a requirement.